data/init.sql

```diff
@@ -182,4 +175,20 @@ CREATE TABLE IF NOT EXISTS comments(
 	created_at TIMESTAMP,
 	updated_at TIMESTAMP,
 	FOREIGN KEY(user_id) REFERENCES users(id)
+CREATE TABLE IF NOT EXISTS course_section_students(
+	user_id INT(11) UNSIGNED NOT NULL,
+	section_id INT(11) UNSIGNED,
+	FOREIGN KEY (user_id) REFERENCES users(id),
```

Comment:
hey this syntax is invalid 🤔

Reply:
oups, I think that's from resolving merge conflicts, sorry bout that